### PR TITLE
add blog -ds run in windows english version

### PR DESCRIPTION
**Pull request by Tianqi-Dotes**
add blog -ds run in Windows OS English version
will add the video element after the video published

### CODE REVIEW

File: blog/en-us/DS_run_in_windows.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Set-up DolphinScheduler Development Environment and Run Source Code on Windows OS
+
+1. ## Download Source Code
+
+   Official Website：https://dolphinscheduler.apache.org/en-us/index.html
```
Comment (ruanwenjun) [inline blog/en-us/DS_run_in_windows.md:5]:
```suggestion
   Official Website: https://dolphinscheduler.apache.org/en-us/index.html
```

File: blog/en-us/DS_run_in_windows.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+   1. Download zookeeper https://www.apache.org/dyn/closer.lua/zookeeper/zookeeper-3.6.3/apache-zookeeper-3.6.3-bin.tar.gz
+
+   2. Unzip 'apache-zookeeper-3.6.3-bin.tar.gz'
+
+   3. Create 'data' and 'log' folder under zookeeper directory
+
+   4. Copy the 'zoo_sample.cfg' file in the 'conf' directory, rename it to 'zoo.cfg' and modify the data and log directory settings:
+
+      ```shell
```
Comment (ruanwenjun) [inline blog/en-us/DS_run_in_windows.md:21]: It's better to use `~~~` here, consistent with following

File: blog/en-us/DS_run_in_windows.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      dataDir=I:\\setup\\apache-zookeeper-3.6.3-bin\\data
+      dataLogDir=I:\\setup\\apache-zookeeper-3.6.3-bin\\log
+      ```
+
+   5. Run the 'zkServer.cmd' in the bin directory, then run 'zkCli.cmd' to check the running status. Check zookeeper node information under root directory which shows success installation.
```
Comment (ruanwenjun) [inline blog/en-us/DS_run_in_windows.md:26]: It's better to add `.`  in the end of all sentence like this.

File: blog/en-us/DS_run_in_windows.md (hunk 1 above)
Comment (Tianqi-Dotes) [reply blog/en-us/DS_run_in_windows.md:5]: done

File: blog/en-us/DS_run_in_windows.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+      ~~~java
+      zookeeper.quorum=localhost:2181
+      ~~~
+
+   6. Add standard output in the 'logback-worker.xml', 'logback-master.xml' and 'logback-api.xml'
+
+      ~~~xml
+      <root level="INFO">
+          <appender-ref ref="STDOUT"/>  <!-- 添加控制台输出 -->
```
Comment (QuakeWang) [inline blog/en-us/DS_run_in_windows.md:56]: There is no translation of this sentence.

File: blog/en-us/DS_run_in_windows.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+          <appender-ref ref="APILOGFILE"/>
+          <appender-ref ref="SKYWALKING-LOG"/>
+      </root>
+      ~~~
+
+
+7. Run MasterServer, run the main method of 'org.apache.dolphinscheduler.server.master.MasterServer' and VM Options need to be set are:
```
Comment (QuakeWang) [inline blog/en-us/DS_run_in_windows.md:63]: This sentence appears to be the result of machine translation. It is recommended to make a readjustment.

File: blog/en-us/DS_run_in_windows.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+   -Dlogging.config=classpath:logback-master.xml -Ddruid.mysql.usePingMethod=false
+   ~~~
+
+8. Run WorkerServer, run the main method of 'org.apache.dolphinscheduler.server.worker.WorkerServer' and VM Options need to be set are:
+
+   ~~~java
+   -Dlogging.config=classpath:logback-worker.xml -Ddruid.mysql.usePingMethod=false
+   ~~~
+
+9. Run ApiApplicationServer, run the main method of 'org.apache.dolphinscheduler.api.ApiApplicationServer' and VM Options need to be set are:
```
Comment (QuakeWang) [inline blog/en-us/DS_run_in_windows.md:75]: Same as the previous recommendation.

File: blog/en-us/DS_run_in_windows.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+      ~~~
+
+
+7. Run MasterServer, run the main method of 'org.apache.dolphinscheduler.server.master.MasterServer' and VM Options need to be set are:
+
+   ~~~java
+   -Dlogging.config=classpath:logback-master.xml -Ddruid.mysql.usePingMethod=false
+   ~~~
+
+8. Run WorkerServer, run the main method of 'org.apache.dolphinscheduler.server.worker.WorkerServer' and VM Options need to be set are:
```
Comment (QuakeWang) [inline blog/en-us/DS_run_in_windows.md:69]: Same as the previous recommendation.

File: blog/en-us/DS_run_in_windows.md (hunk 1 above)
Comment (kezhenxu94) [reply blog/en-us/DS_run_in_windows.md:5]: Find and replace all `：` to `:`

File: blog/en-us/DS_run_in_windows.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+3. ## Set-up Backend Environment
+
+   1. Create MYSQL database for debugging, named as 'dolphinschedulerTest'
+
+   2. Import Maven project to IDEA, find 'pom.xml' under the root directory and modify the scope of 'mysql-connector-java' to compile
+
+   3. Modify the 'datasource.properties' of module 'dolphinscheduler-dao'
+
+      ~~~java
```
Comment (kezhenxu94) [inline blog/en-us/DS_run_in_windows.md:36]: This is not `java` syntax code snippet

File: blog/en-us/DS_run_in_windows.md (hunk 2 above)
Comment (kezhenxu94) [reply blog/en-us/DS_run_in_windows.md:21]: It’s not `shell` syntax, should be `properties`

File: blog/en-us/DS_run_in_windows.md (hunk 3 above)
Comment (Tianqi-Dotes) [reply blog/en-us/DS_run_in_windows.md:26]: done

File: blog/en-us/DS_run_in_windows.md (hunk 4 above)
Comment (Tianqi-Dotes) [reply blog/en-us/DS_run_in_windows.md:56]: done

File: blog/en-us/DS_run_in_windows.md (hunk 8 above)
Comment (Tianqi-Dotes) [reply blog/en-us/DS_run_in_windows.md:36]: Thanks, have changed to no type (normal) code snippet

File: blog/en-us/DS_run_in_windows.md (hunk 2 above)
Comment (Tianqi-Dotes) [reply blog/en-us/DS_run_in_windows.md:21]: done

**Comment by zhongjiajie** [comment]
Hi @Tianqi-Dotes  are you still WIP in this PR, I see we have some unsolve reviews here.

**Comment by zhongjiajie** [comment]
I add the label `help wanted` for this PR, if some who interested in this, feel free to take it. Hope you don't mind @Tianqi-Dotes

**Comment by linjicong** [comment]
I can help @zhongjiajie

**Comment by zhongjiajie** [comment]
> I can help @zhongjiajie

Great! I assigned this to you. cc @Tianqi-Dotes

**Comment by linjicong** [comment]
#641 change to this pr

**Comment by zhongjiajie** [comment]
Close due to #641, thanks @linjicong , cc @Tianqi-Dotes

**Pull request closed without merging**